src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,376 @@
+| OtherAcquisitionParameters | OPTIONAL              | [string][]               | Description of other relevant image acquisition parameters.                                                                                                                                                                                                                                                                                                                                                                                                              |
+
+#### Sample
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+
+Note: For `BodyPart`, we removed: "Corresponds to DICOM Tag 0018, 0015 `Body Part Examined`. For example: `"BRAIN"`."
+from the description, and replace with
+"from [DICOM Body Part Examined](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html#chapter_L) (for example: `"BRAIN".`)".
+It will require changes to `src/schema/metatata/BodyPart.yaml` and `src/04-modality-specific-files/09-positron-emission-tomography.md`.
+-->
+| **Key name**                | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| --------------------------- | --------------------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BodyPart                    | RECOMMENDED           | [string][]                             | Body part of the organ / body region scanned. From [DICOM Body Part Examined](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html#chapter_L) (for example: `"BRAIN"`)".                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Good question!

As far as I know, there is no similar field in the current BEP032 proposal, maybe @SylvainTakerkart  or @JuliaSprenger  can confirm. Please let us know if we need to harmonize, thanks!

BodyPart was chosen to fit with BEP025 as well as PET who are also using the field.

For PET and microscopy, there is a slight difference in the second sentence of the definition because PET is extracted from the DICOM field (I think) and not microscopy. This will have to be dealt with after community review with yaml file but did not seem to be an issue (see previous comment: https://github.com/bids-standard/bids-specification/pull/881#issuecomment-923399363)

Reply:
I agree with @mariehbourget on this. I would like to add that for microscopy in particular, perhaps both BodyPart and BodyPartDetails should be RECOMMENDED - I can think of several instances where this could be of consequence, not the least of which are biopsy (or autopsy) samples for use in multisubject and/or multimodal analyses.

Reply:
Thank you @pjtoussaint for the feedback.
We are in favor of making `BodyPartDetails` RECOMMENDED as well for microscopy.
Integrated in 60ecb3d.